pkg/cloudevents/event_marshal_test.go

```diff
@@ -489,19 +489,6 @@ func TestUnmarshal(t *testing.T) {
 	}
 }
 
-func TestUnmarshal_nilEvent(t *testing.T) {
```

Comment:
why delete the test?

Reply:
It is not testing anything in this package - it's verifying the documented behavior of json.Unmarshal()
I'm hoping the json package has it's own tests :)